src/setup.rs

```diff
@@ -90,19 +90,21 @@ pub(crate) fn initial_state(config: config::SpotifydConfig) -> main_loop::MainLo
         }
     }
 
-    let credentials_provider = if let Some(credentials) =
-        get_credentials(&cache, &username, &password)
-    {
-        CredentialsProvider::SpotifyCredentials(credentials)
-    } else {
-        info!("no usable credentials found, enabling discovery");
-        let discovery_stream = librespot_discovery::Discovery::builder(config.device_name.clone())
-            .device_type(device_type)
-            .port(zeroconf_port)
-            .launch()
-            .unwrap();
-        discovery_stream.into()
-    };
+    let credentials_provider =
+        if let Some(credentials) = get_credentials(&cache, &username, &password) {
+            CredentialsProvider::SpotifyCredentials(credentials)
+        } else {
+            info!("no usable credentials found, enabling discovery");
+            info!("device_id {}", session_config.device_id.clone());
```

Comment:
If we want to add it for everyone, I would change that a bit. Since this log doesn't add much value for normal users, I think it should be debug, so you have to opt in (`--verbose`) to see it.

```suggestion
            debug!("Using device id '{}'", session_config.device_id);
```

Reply:
Done